src/app/base/components/TagSelector/TagSelector.tsx

```diff
@@ -31,6 +31,9 @@ export type Props = {
   disabledTags?: Tag[];
 };
 
+// Give an explicit name to the text input so we can exclude it from form submission
+export const TAG_SELECTOR_INPUT_NAME = "tagSelectorInput" as const;
```

Comment:
```suggestion
export const TAG_SELECTOR_INPUT_NAME = "tagSelectorInput";
```
`as const` is redundant here. Strings are immutable in JavaScript, there’s no need to use as const for them.